src/collections/programs/hacktoberfest-2020/index.mdx

```diff
@@ -87,7 +87,7 @@ import hacktoberfest2020 from "./hacktoberfest2021.png";
         <a href="https://docs.google.com/presentation/d/1Wc5ALdn-G3fADJ8I6nJlyOGhV2XHVprYPqDvkZ1MEqY/edit#" target="_blank">slides</a>, <a href="https://youtu.be/67iy2JEp4Ss" target="_blank">recording</a>
       </td>
       <td>
-        <a href="/community/members/vijay-cherukuri">Vijay Cherukuri</a>
+        <a href="https://layer5.io/community/members/vijay-cherukuri">Vijay Cherukuri</a>
```

Comment:
I doubt if this is needed. The use of `/` before the path ensures that the path is from the root of the site.

Reply:
I tried clicking each one of them but none of them worked. so I added "https://layer5.io"